articles/embedded.md

```diff
@@ -0,0 +1,309 @@
+---
+layout: default
+title: Embedded ROS 2.0
+abstract:
+  This article captures requirements, design ideas and related works on a tiny ROS 2.0 stack for microcontrollers.
+published: true
+author: 'In alphabetic order: [Adam Dąbrowski](https://github.com/adamdbrw), [Borja Outerelo](https://github.com/BorjaOuterelo), [Ingo Lütkebohle](https://github.com/iluetkeb), [Ralph Lange](https://github.com/ralph-lange), [Víctor Mayoral Vilches](https://github.com/vmayoral), ... and many more from Embedded ROS 2 Interest Group. Please feel free to add your name.'
+---
+
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+While standard ROS 2 can run on Linux-class embedded devices, smaller 
+microcontrollers (MCU) are still a challenge. For example, ARM Cortex-M3 
+and -M4 class MCUs are popular for motion control and sensor integration, 
+but have little RAM and storage. ARM Cortex-M3 is widely used in many 
+sensors due to its optimized power consumption capabilities and ultra low 
+power modes. These microcontrollers are often running with small batteries, 
+e.g. for environmental sensors or small consumer robots, which requires
+efficient power management. We aim to address these devices through a 
+combination of specialized, ROS 2-interoperable stack as well as the use
```

Comment:
Show your reasoning for *not* addressing these devices through improvements in rmw/rcl.

Reply:
At this stage "specialized" doesn't mean it couldn't be rmw/rcl, but also it doesn't *have* to be rmw/rcl.